huey_monitor/tasks.py

```diff
@@ -19,13 +23,10 @@ def store_signals(signal, task, exc=None):
     task_id = uuid.UUID(task.id)
     logger.info('Store Task %s signal %r', task_id, signal)
 
-    task_model_instance, created = TaskModel.objects.get_or_create(
-        task_id=task_id,
-        defaults={'name': task.name}
-    )
-
     signal_kwargs = {
-        'task': task_model_instance,
+        'domain': socket.getfqdn(),
```

Comment:
This can be expensive and involve a DNS query. Do we really want to do it on every signal? If so, then we should cache it in process.

Reply:
Yes, you are absolutely right.
Think i will move it to module level.

Reply:
updated